go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/husniadil/uuid
 go 1.15
 
```

Comment:
The Go module version specified is `go 1.15`, which is quite outdated. Consider updating the Go version to a more recent stable release (e.g., 1.20 or later) to take advantage of performance improvements, security patches, and new language features. This will also help ensure compatibility with newer dependencies.